skins/elementfinance.scss

```
@@ -0,0 +1,16 @@
+.element-finance {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.elementfinance {
```

---

@tinahaibodi Could you also update this file, please? Thank you!